core/App/ehsm_core_test.cpp

```diff
@@ -956,11 +956,12 @@ void test_GenerateQuote_and_VerifyQuote()
     printf("============test_GenerateQuote_and_VerifyQuote start==========\n");
     char challenge[32] = "challenge123456";
     char nonce[16] = "nonce123456";
-
+    char mr_signer[65] = "c30446b4be9baf0f69728423ea613ef81a63e72acf7439fa0549001fd5482835";
```

Comment:
need a note for original data

Reply:
Ok, I'll add note for it.

Reply:
mr_signer -> mrsigner, same with the below file.

Reply:
Ok, I'll change them